### PR TITLE
feat: roadmap + updates

### PR DESCRIPTION
**Pull request by 0ubbe**
- Add **Roadmap** page in "Getting started"
- Reword "Passwordless" to "Token-based auth" 
- Reword "External auth" to "Credentials"

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,15 +16,9 @@
...
     "paths": {
-      "@/components/*": [
-        "components/*"
-      ],
-      "@/utils/*": [
-        "utils/*"
-      ],
-      "@/data/*": [
-        "pages/data/*"
-      ]
```
Comment (0ubbe) [inline tsconfig.json]: Prettier version issues here 🤔 . Also running the dev server is changing `jsx` to `preserve` rather than `react-jsx`.

Comment (ndom91) [reply tsconfig.json]: Yeah i had that too in my latest PR, just discarded the changes to this file. But if we're all having it then it's something we'll have to look into :/

**Review by ndom91 — APPROVED** [review]
But otherwise LGTM! I'll take a look at the prettier tsconfig changes locally from my side too

File: tsconfig.json (hunk 1 above)
Comment (0ubbe) [reply tsconfig.json]: I think we actually should add `"react/react-in-jsx-scope": "off"` to the lint config to prevent this warning 💭

**Pull request merged**